doc/source/upstream-bug-fixes.rst

```diff
@@ -5,17 +5,35 @@
 =============================================================
 
 +-------------------------------------------------------------------------------------------------------------+
+|:Upstream bug: :mysqlbug:`86464` - INFORMATION_SCHEMA.TABLES missing performance_schema.session&global_var...|
```

Comment:
The upstream bug here is not fixed, but only worked around - probably better to remove its mention